### PR TITLE
New pages: DOMMatrix.invert()

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/openwebdocs/project/issues/214

### CODE REVIEW

File: files/en-us/web/api/dommatrix/invertself/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "DOMMatrix: invertSelf() method"
+short-title: invertSelf()
+slug: Web/API/DOMMatrix/invertSelf
+page-type: web-api-instance-method
+browser-compat: api.DOMMatrix.invertSelf
+---
+
+{{APIRef("Geometry Interfaces")}}{{AvailableInWorkers}}
+
+The **`invertSelf()`** method of the {{domxref("DOMMatrix")}} interface inverts the original matrix. If the matrix cannot be inverted, the new matrix's components are all set to `NaN` and its {{domxref("DOMMatrix.is2D", "is2D")}} property is set to `false`. The original matrix is not changed.
```
Comment (wbamberg) [inline files/en-us/web/api/dommatrix/invertself/index.md:11]: Are you planning to write a page for `is2D`? Currently it's not a link in https://pr37172.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/DOMMatrixReadOnly#instance_properties.

Comment (estelle) [inline files/en-us/web/api/dommatrix/invertself/index.md:11]:
```suggestion
The **`invertSelf()`** method of the {{domxref("DOMMatrix")}} interface inverts the original matrix. If the matrix cannot be inverted, the new matrix's components are all set to `NaN` and its {{domxref("DOMMatrix.is2D", "is2D")}} property is set to `false`.
```

Comment (estelle) [reply files/en-us/web/api/dommatrix/invertself/index.md]: > Are you planning to write a page for `is2D`? Currently it's not a link in https://pr37172.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/DOMMatrixReadOnly#instance_properties.

That page is in PR https://github.com/mdn/content/pull/37169

Comment (estelle) [reply files/en-us/web/api/dommatrix/invertself/index.md]: Thanks for the catch. I missed it when editing the 'save as' from the dommatrixreadonly version of the inversion property.

**Review by wbamberg — APPROVED** [review]
👍 thanks @estelle !

**Pull request merged**